staging/src/k8s.io/client-go/tools/leaderelection/leaderelection.go

```diff
@@ -214,6 +215,7 @@ func (le *LeaderElector) acquire(ctx context.Context) bool {
 		klog.Infof("successfully acquired lease %v", desc)
 		cancel()
 	}, le.config.RetryPeriod, JitterFactor, true, ctx.Done())
+	leaderGauge.WithLabelValues(le.config.Name).Set(1.0)
```

Comment:
Shouldn't we check the value of succeeded prior to toggling the gauge? I'm not super familiar with this code, but at least according to the comment for this function, there seems to be a situation where we did not succeed in acquiring the lease.

Reply:
Good point. If succeeded is false we are about to exit but no point in putting a blip on the graphs.